src/pages/HomePage/HomePage.js

```diff
@@ -8,6 +9,34 @@ import TextField from "@material-ui/core/TextField";
 import Button from "@material-ui/core/Button";
 
 const HomePage = () => {
+  const [state, setState] = useState({ email: "" });
```

Comment:
We may want to use a better name than `state` and something like `userInfo` and `SetUserInfo`. But it might be thought of after we know what exactly this page will do.